src/Fragment/FragmentConfig.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Fragment;
+
+class FragmentConfig
+{
+    /**
+     * @var string
+     */
+    private $controller;
+
+    /**
+     * @var string
+     */
+    private $renderer;
+
+    /**
+     * @var array
+     */
+    private $options;
+
+    /**
+     * Constructor.
+     *
+     * @param string $controller
+     * @param string $renderer
+     * @param array  $options
+     *
+     * @see \Symfony\Component\HttpKernel\Fragment\FragmentHandler::render()
+     */
+    public function __construct(string $controller, string $renderer = 'inline', array $options = [])
```

Comment:
Sorry for changing my mind, but `$renderer` is wrong. It suggests that one has to pass a `new FragmentRenderer()` object. We should use `$renderingStrategy` as used in Symfony.

Reply:
It's called `renderer` in Symfony! https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/Fragment/FragmentHandler.php#L65

Reply:
True.